### ISSUE TITLE
Number of PRs label should be mapped to the respective combobox

### ISSUE THREAD

**Issue by claudiahub** [backstage/backstage#11207]
## Expected Behavior

Label "Number of PRs" should be mapped to the respective combo box.
Screen reader should announce the label i.e., `Number of PRs clickable 20 menu button submenu`. 

## Actual Behavior

1. Navigate to `Home` in side nav.
2. Go to main section of the page and select a link from the name column.
3. In the next page, select link under parent component text. A new page opens up.
4. Navigate to `Github` section and see to the number of PRs combo box and press F12 to open developer tool
5. Verify in the Dom, whether the label is properly mapped to the combo box or not. 

<img width="1792" alt="Schermata 2022-05-02 alle 17 00 28" src="https://user-images.githubusercontent.com/54085737/166257144-7a2bf74a-ee99-4268-901f-ca812aba65a0.png">

## Context

When the focus is on number of PRs combo box, screen reader announces `Clickable 20 menu button submenu`.

User Impact: Assistive technology uses the label to help ensure people are able to interact with the input fields. This includes people who use screen readers, people who use voice navigation, and people who are neurodiverse who use browser plugins

A11y Conformance Level: A

WCAG Success Criteria: 1.3.1

## Environment

Browser Information: 
Google Chrome v100.0.4896.127
Firefox 99.0.1
Safari 15.4

**Comment by kissmikijr** [comment]
Hey! Thank you for opening this issue! This is actually one of our plugins (Roadie) which we keep in a different repository. Could you please open an issue for us in our plugins repo? Thanks!

https://github.com/RoadieHQ/roadie-backstage-plugins/tree/main/plugins/frontend/backstage-plugin-github-pull-requests

**Comment by claudiahub** [comment]
Hey @kissmikijr thanks for clarifying on this!
I'll create an issue for the Roadie plugin.
